### PR TITLE
Completed Hashing-2 programs

### PR DESCRIPTION
**Pull request by Siri-isha**

### CODE REVIEW

File: LongestPalindrome.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//Time Complexity : O(n)
+//Space Complexity: O(n)
```
Comment (ishika2741) [inline LongestPalindrome.java]: check space complexity

**Review by ishika2741 — COMMENTED** [review]
Good work!

**Comment by ishika2741** [comment]
Good work!

File: LongestPalindrome.java (hunk 1 above)
Comment (Siri-isha) [reply LongestPalindrome.java]: It must be O(1), I have corrected it now. Thank you

**Pull request closed without merging**